openapi/nexus.json

```diff
@@ -3947,6 +3947,16 @@
             "style": "simple"
           }
         ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/InstanceStart"
+              }
+            }
+          },
+          "required": true
+        },
```

Comment:
I'm just trying to frame this in my head... so the user does a `POST .../instances/:instance/start` with `{ only_provision: true }` as the body and the instances is provisioned but still isn't in the start state yet. So to start it, the post must be re-submitted sans the `only_provision` flag?

Would there be a downside to introducing `POST /provision` as an alternative to the `only_start` body?

Reply:
I'm with Justin, I think it's weird to say "start, but like... not really."

Reply:
Sure, honestly this was a wart I did want to come back to anyways.

Reply:
done.